src/utils/c_string_util.h

```diff
@@ -9,6 +9,10 @@ char *strncpy_or_truncate(char *dest, const char *src, size_t n);
 // strncpy() that guarantees null-termination by aborting on overflow
 char *strncpy_or_abort(char *dest, const char *src, size_t n);
 
+// libgen.h
+char *omf_basename(char *s);
```

Comment:
These would probably go in like a fs_util.c?

Reply:
Well, they are purely string operations, but maybe that's reasonable. Anyway, need to cleanup the implementation first ..